internal/ante/min_commission_test.go

```diff
@@ -0,0 +1,8 @@
+package ante_test
+
+import "testing"
+
+func TestMinCommissionAnteHandler(t *testing.T) {
+	// simapp.New(t.TempDir())
```

Comment:
Want to enable this?

Reply:
haven't actually wrote a test for this so it was disabled, will write a test now

Reply:
added tests